framework/src/test/java/io/github/kgress/scaffold/webelement/HiddenElementTests.java

```diff
@@ -0,0 +1,59 @@
+package io.github.kgress.scaffold.webelement;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+import io.github.kgress.scaffold.BaseComponent;
+import io.github.kgress.scaffold.BaseUnitTest;
+import io.github.kgress.scaffold.HiddenElement;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+@Slf4j
+public class HiddenElementTests extends BaseUnitTest {
+
+  class TestComponent extends BaseComponent {
+
+    @HiddenElement
+    private final TestBaseWebElement hiddenThing = new TestBaseWebElement(
+        By.cssSelector(".hiddenElement"));
+
+    private final TestBaseWebElement notHiddenThing = new TestBaseWebElement(
+        By.cssSelector(".notHidden"));
+
+    public TestComponent() {
+      super();
+      applyFieldAnnotations();
+    }
+
+    public TestBaseWebElement getHiddenThing() {
+      return hiddenThing;
+    }
+
+    public TestBaseWebElement getNotHiddenThing() {
+      return notHiddenThing;
+    }
```

Comment:
Small detail on this TestComponent, can add @Getter at the class level and strip out these methods.